CONTRIBUTING.md

```diff
@@ -212,7 +209,7 @@ the existing plugins for examples, and see additional advice in
 In this repository, `master` represents the current state of the website, and
 should constantly be deployed. Branch off of this branch for local development.
 
-If you have write access to the GitHub repository, please follow the following
+If you have to write access to the GitHub repository, please follow the following
```

Comment:
"If you have write access" is correct, not "to write."

Write access is a concept in computing that refers to the permission of a user to "write" data to memory. Compare it with "read access," etc.

Grammarly and other spellcheckers do not understand this, but human readers of documentation should.

Reply:
Thanks for your comment, I resolved it back to original